shakedown/cli/main.py

```diff
@@ -336,4 +383,15 @@ def pytest_sessionfinish(self, session, exitstatus):
 
     exitstatus = imported['pytest'].main(opts, plugins=[shakedown()])
 
+    if args['provisioner_url']:
+      echo('Cleaning up...', d='step-maj')
+      echo('Destroying cluster...', d='step-min', n=False)
+      response = requests.delete("{}/api/v1/vm/{}".format(args['provisioner_url'], args['cluster_id']))
+      if ('ok' in response) and (response['ok']):
```

Comment:
There is retry login on the back-end (vmpooler will retry deleting AWS `FAILED_DELETE` clusters) but making multiple requests won't do anything (the request to the vmpooler API is just saying "move this from `running` to `destroy` queue).